language.md

```diff
@@ -475,6 +469,37 @@ Performs multiple collections at once and converts the results into an object.
 This can be very useful to share a `Where` condition while collecting multiple
 pieces of information.
 
+- [_behaviour_] `` ` `` _collector_ `` ` ``
+
+This allows using optional values in other collectors. For instance, suppose
+there is an optional number and the minimum is desired, one could write: `` `
+Min x? ` ``.
+
+There are special _behaviours_ for how to handle records with missing data:
+
+- the unspecified behaviour is to simply drop empty optionals and proceed with
+  the collector as normal. This is equivalent to ``Where x != ` ` Min x Default
+  1234 ``; since `x` will never have the empty optional, the default will never
+  be used.
+- `OnlyIf All` requires that no empty input makes it to the collector; if any
+  optional input is found, the output from the collector is replaced by the
+  empty optional. So, if ``v = OnlyIf All `Min x?` `` was given `` `3` and `` ` `
+  ``, it would produce `` v == ` ` ``.
+- `OnlyIf Any` requires that one non-empty empty input makes it to the
```

Comment:
what is a 'non-empty empty input'? or is that a typo

Reply:
Typo.